plugins/jobs/src/js/components/JobDeleteModal.js

```diff
@@ -26,9 +27,9 @@ const ModalMessage = ({ jobId, stopCurrentJobRuns }) => {
 
   return (
     <div>
-      <h2 className="text-danger text-align-center flush-top">
+      <Trans render="h2" className="text-danger text-align-center flush-top">
         {StringUtil.capitalize(UserActions.DELETE)} Job
```

Comment:
if `UserActions.DELETE` is a constant what is the value of the variable here? I know it was like this already but for translation including the string literal or moving the `StringUtil.capitalize(UserActions.DELETE)` to a const to be translated with `t` or `i18nmark` _could_ be better for translating.

Reply:
I think we would handle that in the ticket associated with non-jsx/constant resources?

Reply:
I disagree. I think the translatable string we want in the messages.json file includes the word "Delete" and shouldn't be translated separately.